pkg/operator/ceph/pool/status.go

```diff
@@ -51,6 +51,7 @@ func updateStatus(client client.Client, poolName types.NamespacedName, status ce
 		logger.Warningf("failed to set pool %q status to %q. %v", pool.Name, status, err)
 		return
 	}
+	pool.Status.ObservedGeneration = pool.ObjectMeta.Generation
```

Comment:
This is following the previous pattern that didn't work, did you just miss removing this one?

Reply:
Yes this file was not saved, thanks